Sources/TuistLoader/Models+ManifestMappers/CopyFileElement+ManifestMapper.swift

```diff
@@ -18,38 +18,20 @@ extension XcodeGraph.CopyFileElement {
     ) throws -> [XcodeGraph.CopyFileElement] {
         func globFiles(_ path: AbsolutePath) throws -> [AbsolutePath] {
             if FileHandler.shared.exists(path), !FileHandler.shared.isFolder(path) { return [path] }
-
+            
+            if !path.pathString.isGlobComponent {
+                return [path]
+            }
             let files = try FileHandler.shared.throwingGlob(AbsolutePath.root, glob: String(path.pathString.dropFirst()))
                 .filter(includeFiles)
 
-            if files.isEmpty {
-                if FileHandler.shared.isFolder(path) {
-                    logger.warning("'\(path.pathString)' is a directory, try using: '\(path.pathString)/**' to list its files")
-                } else {
-                    // FIXME: This should be done in a linter.
-                    logger.warning("No files found at: \(path.pathString)")
-                }
+            if files.isEmpty && FileHandler.shared.isFolder(path) {
```

Comment:
Similar comment here regarding the `path` being a glob at this point.

Reply:
Fixed this as well